addons/point_of_sale/tools/posbox/configuration/wireless_ap.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 FORCE_HOST_AP="${1}"
-WIRED_IP=$(ifconfig eth0 | grep "inet addr" | awk -F: '{print $2}' | awk '{print $1}';)
+WIRED_IP=$(python3 -c "import netifaces as ni; print(ni.ifaddresses('eth0').get(ni.AF_INET) and ni.ifaddresses('eth0')[ni.AF_INET][0]['addr'] or '')")
```

Comment:
the new `ifconfig` is `ip address`

Reply:
I liked python better as it won't change if the output changes somehow.  We could also check ifconfig on "inet "

Reply:
and maybe echo does not print